src/benchmarks/micro/libraries/System.IO.Compression/CompressionStreamPerfTestBase.cs

```diff
@@ -59,7 +59,7 @@ public void Compress()
         {
             CompressedFile.CompressedDataStream.Position = 0; // all benchmarks invocation reuse the same stream, we set Postion to 0 to start at the beginning
 
-            var compressor = CreateStream(CompressedFile.CompressedDataStream, level);
+            using var compressor = CreateStream(CompressedFile.CompressedDataStream, level);
```

Comment:
It's less of an issue, but https://github.com/dotnet/performance/blob/ccbeb4ed946e0a7bb676c58f6b5841740c616873/src/benchmarks/micro/libraries/System.IO.Compression/CompressionStreamPerfTestBase.cs#L71 should also be disposing of this.